packages/jupyter-ai/jupyter_ai/handlers.py

```diff
@@ -157,24 +158,39 @@ async def get(self, *args, **kwargs):
         res = super().get(*args, **kwargs)
         await res
 
+    def get_current_user(self) -> ChatUser:
+        """Retrieves the current user. If collaborative mode is disabled, one
+        is synthesized from the login."""
+        collaborative = self.config.get("LabApp", {}).get("collaborative", False)
+
+        if collaborative:
+            return ChatUser(**asdict(self.current_user))
+        
+        
+        login = os.getlogin()
```

Comment:
Are we supposed to use `getpass.getuser()` here?
https://docs.python.org/3/library/os.html#os.getlogin